src/nvim/os/win_defs.h

```diff
@@ -7,11 +7,14 @@
 
 // winsock2.h must be first to avoid incompatibilities
 // with winsock.h (included by windows.h)
+
+// uncrustify:indent-off
 #include <winsock2.h>
-#include <windows.h>
-#include <sys/stat.h>
+// uncrustify:indent-on
```

Comment:
this isn't indent-related. is `uncrustify:indent-off` the generic "disable uncrustify" flag? If so we should rename it to `uncrustify:off`

Reply:
It's like you read my mind :)